conans/test/utils/tools.py

```diff
@@ -605,7 +605,7 @@ def alias(self, source, target):
         source = RecipeReference.loads(source)
         target = target.split("/", 1)[1]
         self.run(f"new alias -d name={source.name} -d version={source.version} "
-                 f"-d target={target} -f")
+                 f"-d target={target} -d revision_mode=hash -f")
```

Comment:
This should be able to be defaulted (maybe even removed from the ``alias`` template)

Reply:
This implementation won't accept default in templates, if we think we need them then I can make something different...